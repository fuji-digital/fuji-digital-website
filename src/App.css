.header {
  padding: 1.5rem 2rem;
  display: flex;
  align-items: flex-start;
  justify-content: center;

  @media screen and (min-width: 768px) {
    justify-content: normal;
    padding: 1rem 3rem;
  }
}

.pageStyling {
  position: relative;
  overflow-x: hidden;
}

.body {
  padding: 0rem 2.5rem 2.5rem 2.5rem;

  @media screen and (min-width: 768px) {
    padding: 0rem 2rem 2rem 2rem;
  }
}

.blobStyling {
  position: absolute;
  right: 0;
  top: 7rem;
  z-index: -1;
  height: 10rem;
  @media screen and (min-width: 768px) {
    top: -1rem;
    height: 30rem;
    right: -3rem;
  }
}

.cardLayout {
  display: flex;
  flex-direction: column;
  @media screen and (min-width: 768px) {
    flex-direction: row-reverse;
    justify-content: space-around;
  }
}

.designingImage {
  height: 11rem;
  @media screen and (min-width: 768px) {
    height: 30vw;
  }
}

.textContainer {
  @media screen and (min-width: 768px) {
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: flex-start;
  }
}
h1 {
  font-size: 1.5rem;
  font-family: "Poppins", sans-serif;
  color: #ebf0f7;
  line-height: 1.2;
  @media screen and (min-width: 1200px) {
    font-size: 2rem;
  }
}
p {
  font-size: 1rem;
  font-family: "Poppins", sans-serif;
  color: #ebf0f7;
  line-height: 1.2;
  @media screen and (min-width: 1200px) {
    font-size: 1.5rem;
  }
}

.pinkBlob {
  height: 4rem;
  display: block;
  margin: auto;
  @media screen and (min-width: 768px) {
    margin: 1rem 0;
    height: 8vw;
  }
}
