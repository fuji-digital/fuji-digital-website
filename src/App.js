import * as React from "react";
import "./App.css";

import Logo from "./components/Logo/Logo";
import Card from "./components/Card/Card";
import Footer from "./components/Footer/Footer";

import pinkRoundShortFujit from "./images/fujits/pink-round-short-fujit.svg";
import designingImage from "./images/decorative-images/designing.svg";
import pinkPurpleDecorativeBlobs from "./images/decorative-blobs/pink-purple-decorative-blobs.svg";

function App() {
  return (
    <div className="pageStyling">
      <div className="header">
        <Logo version={"white"} />
      </div>
      <img
        src={pinkPurpleDecorativeBlobs}
        alt="A blue and pink blob"
        className="blobStyling"
      />
      <div className="body">
        <Card>
          <div className="cardLayout">
            <img
              src={designingImage}
              alt="Two cartoon people discussing work over a computer"
              className="designingImage"
            />
            <div className="textContainer">
              <h1>Under Construction</h1>
              <p>We are currently working hard on making an amazing website.</p>
              <p>Stay tuned...</p>
              <img
                src={pinkRoundShortFujit}
                alt="A Pink circle with a happy face with its arms in the air"
                className="pinkBlob"
              />
            </div>
          </div>
        </Card>
      </div>
      <Footer />
    </div>
  );
}

export default App;
