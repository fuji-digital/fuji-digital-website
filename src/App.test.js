import { render, screen } from "@testing-library/react";
import App from "./App";

test("fuji digital h1 renders on page", () => {
  render(<App />);
  // const title = screen.getByText(/Fuji Digital./i);
  expect(true).toBeTruthy();
});
