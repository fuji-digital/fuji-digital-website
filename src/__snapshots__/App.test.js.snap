// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App matches snapshot 1`] = `
<div
  className="sc-aYaIB hHTtjJ"
>
  <div
    className="sc-gEvDqW enqdOK"
  >
    <div
      className="sc-dhKdPU hOfZUt"
    >
      <img
        alt="Fuji Digital Logo"
        className="sc-imWZod bLngYe"
        src="fuji-digital-logo.svg"
      />
      <img
        alt="Fuji Digital Text Logo"
        className="sc-jXbVAB kWTNIY"
        data-testid="textLogo"
        src="fuji-digital-text-logo-white.svg"
      />
    </div>
  </div>
  <img
    alt="A blue and pink blob"
    className="sc-eqUzNf hoQbZc"
    src="pink-purple-decorative-blobs.svg"
  />
  <div
    className="sc-fqkwJk cSFBPC"
  >
    <div
      className="sc-kpDprT iiXXq"
    >
      <div
        className="sc-dcJtft hCAjbi"
      >
        <img
          alt="Two cartoon people discussing work over a computer"
          className="sc-iGgVNO wriun"
          src="designing.svg"
        />
        <div
          className="sc-gsFSjX gyFziH"
        >
          <h1>
            Under Construction
          </h1>
          <p>
            We are currently working hard on making an amazing website.
          </p>
          <p>
            Stay tuned...
          </p>
          <img
            alt="A Pink circle with a happy face with its arms in the air"
            className="sc-kAycRU AVlTJ"
            src="pink-round-short-fujit.svg"
          />
        </div>
      </div>
    </div>
  </div>
  <div
    className="sc-jlZhRR dTCUaw"
  >
    <div
      className="sc-dhKdPU hOfZUt sc-cwHqhk hYoyZo"
    >
      <img
        alt="Fuji Digital Logo"
        className="sc-imWZod bLngYe"
        src="fuji-digital-logo.svg"
      />
      <img
        alt="Fuji Digital Text Logo"
        className="sc-jXbVAB kWTNIY"
        data-testid="textLogo"
        src="fuji-digital-text-logo-purple.svg"
      />
    </div>
    <div
      className="sc-jEABJj cPHdFy"
    >
      <div>
        Fuji Digital Ltd.
      </div>
      <a
        href="mailto: contact@fujidigital.co.uk"
      >
        contact@fujidigital.co.uk
      </a>
    </div>
    <div
      className="sc-dAlxHm iQxgGB sc-cPiJYC ifNUnm"
    >
      <a
        href="https://www.linkedin.com/company/fuji-digital-tech/"
        target="blank"
      >
         
        <svg
          height="35"
          viewBox="0 0 24 24"
          width="35"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M12 0c-6.627 0-12 5.373-12 12s5.373 12 12 12 12-5.373 12-12-5.373-12-12-12zm-2 16h-2v-6h2v6zm-1-6.891c-.607 0-1.1-.496-1.1-1.109 0-.612.492-1.109 1.1-1.109s1.1.497 1.1 1.109c0 .613-.493 1.109-1.1 1.109zm8 6.891h-1.998v-2.861c0-1.881-2.002-1.722-2.002 0v2.861h-2v-6h2v1.093c.872-1.616 4-1.736 4 1.548v3.359z"
          />
        </svg>
      </a>
      <a
        href="https://github.com/fuji-digital"
        target="blank"
      >
        <svg
          height="35"
          viewBox="0 0 24 24"
          width="35"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M12 0c-6.627 0-12 5.373-12 12s5.373 12 12 12 12-5.373 12-12-5.373-12-12-12zm2.218 18.616c-.354.069-.468-.149-.468-.336v-1.921c0-.653-.229-1.079-.481-1.296 1.56-.173 3.198-.765 3.198-3.454 0-.765-.273-1.389-.721-1.879.072-.177.312-.889-.069-1.853 0 0-.587-.188-1.923.717-.561-.154-1.159-.231-1.754-.234-.595.003-1.193.08-1.753.235-1.337-.905-1.925-.717-1.925-.717-.379.964-.14 1.676-.067 1.852-.448.49-.722 1.114-.722 1.879 0 2.682 1.634 3.282 3.189 3.459-.2.175-.381.483-.444.936-.4.179-1.413.488-2.037-.582 0 0-.37-.672-1.073-.722 0 0-.683-.009-.048.426 0 0 .46.215.777 1.024 0 0 .405 1.25 2.353.826v1.303c0 .185-.113.402-.462.337-2.782-.925-4.788-3.549-4.788-6.641 0-3.867 3.135-7 7-7s7 3.133 7 7c0 3.091-2.003 5.715-4.782 6.641z"
          />
        </svg>
      </a>
    </div>
    <div
      className="sc-dLNtp iWqaqT"
    >
      Copyright Â© 2024 Fuji Digital Ltd. All rights reserved.
    </div>
    <img
      className="sc-eDPFhE hsWCZb"
      src="white-purple-decorative-blobs.svg"
    />
    <img
      className="sc-eldOKa gyOUUg"
      src="pink-round-tall-fujit.svg"
    />
  </div>
</div>
`;
